src/index.html

```diff
@@ -13,7 +13,31 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
-<script type="text/javascript" src="./main.js"></script>
-</body>
+  <script type="text/javascript" src="./main.js"></script>
+  <div class="container1 border-box" id="qa-block">
```

Comment:
don't use such names as container1, container2 etc, give the elements names that describe their behavior